docs/update-empty-customer-data-from-addresses/README.md

```diff
@@ -0,0 +1,52 @@
+# Update empty customer data from addresses
+
+Tags: Address, Customers
+
+Running when a customer is updated, this task will fill empty customer names and/or phones on accounts by pulling the relevant data from any customer addresses on file. Optionally, the task may be run in manual mode to scan and update all customers.
+
+* View in the task library: [usemechanic.com/task/update-empty-customer-data-from-addresses](https://usemechanic.com/task/update-empty-customer-data-from-addresses)
+* Task JSON, for direct import: [task.json](../../tasks/update-empty-customer-data-from-addresses.json)
+* Preview task code: [script.liquid](./script.liquid)
+
+## Default options
+
+```json
+{
+  "update_customer_first_and_last_name__boolean": null,
+  "update_customer_phone__boolean": null,
+  "enable_manual_runs_to_scan_all_customers__boolean": false
+}
+```
+
+[Learn about task options in Mechanic](https://docs.usemechanic.com/article/471-task-options)
+
+## Subscriptions
+
+```liquid
+{% if options.enable_manual_runs_to_scan_all_customers__boolean %}
+  mechanic/user/trigger
+  mechanic/shopify/bulk_operation
+{% else %}
+  shopify/customers/update
+{% endif %}
+```
+
+[Learn about event subscriptions in Mechanic](https://docs.usemechanic.com/article/408-subscriptions)
+
+## Documentation
+
+Running when a customer is updated, this task will fill empty customer names and/or phones on accounts by pulling the relevant data from any customer addresses on file. Optionally, the task may be run in manual mode to scan and update all customers.
+
+This task is a combination of two separate merchant requests, the first desiring a way to backfill empty customer registration names once initial orders were made, and the second which noted that customer phone numbers were only set when they opted in to receive SMS updates during checkout.
+
+Submit your [task requests](https://mechanic.canny.io/task-requests) for consideration by the Mechanic community, and they may be chosen for development and inclusion in the [task library](https://tasks.mechanic.dev/)!
```

Comment:
I think this paragraph deserves a place in _every_ readme, so I suggest (1) removing it from this pr, and (2) filing another pr (@mattsodomsky, do you want to do this part?) that adds this line to the markdown construction:

https://github.com/lightward/mechanic-tasks/blob/edb7af5d27878d622db0d2b030a63e2a497b8310/lib/util.js#L157-L161

Reply:
Works for me, I'll remove from PR in favor of a global doc insertion.

Reply:
Yep, I got this part.